pages/association/board.md

```diff
@@ -0,0 +1,55 @@
+---
+title: 'Board of Directors'
+description: 'The board makes decisions regarding the Comunica Association'
+---
+
+The [Comunica Association](/association/) has a Board of Directors
+that makes decisions with respect to the Comunica Association.
+This page describes details on who are the members of this board, what it does, and how it works.
+
+## Members
+
+* [Ruben Taelman](https://www.rubensworks.net/) [(IDLab, Ghent University – imec)](https://knows.idlab.ugent.be/) - Code maintainer and [BDFL](https://en.wikipedia.org/wiki/Benevolent_dictator_for_life)
+* [Pieter Colpaert](https://pietercolpaert.be/) [(IDLab, Ghent University – imec)](https://knows.idlab.ugent.be/) - Management contributor
+
+## Goals
+
+The Board of Directors makes decisions concerning the following topics:
+
+* Determine long-term goals via the [roadmap](/roadmap/).
+* Suggest priorities of issues to the maintainers for short-term development via the [project boards](https://github.com/orgs/comunica/projects).
+* Coordinate future of the Comunica Association
+
+Furthermore, board meetings can be used to evaluate the maintenance and development of Comunica and its related dependencies,
+which includes development by externals via the [Bounty Program](/association/bounties/).
+
+## Becoming a Board Member
+
+There are two ways to become a Board Member:
+
+1. Become a financial contributor via [Open Collective](https://opencollective.com/comunica-association) of at least the Silver tier.
+2. Become a regular [contributor](/contribute/) in any other way, with a dedication of at least two hours per week.
+
+## Decision-making Process
+
+At least once every quarter, the board virtually meets for a board meeting.
+Not all members are required to be present at each meeting.
+The chair is expected to prepare an agenda ahead of time,
+which should contain points raised by the board members.
+A meeting may be skipped if there are no objections from members.
+
+The chair is appointed by the board members, and may be changed at any time through a decision.
+The title of [BDFL](https://en.wikipedia.org/wiki/Benevolent_dictator_for_life) is reserved for one person,
+and can only be passed on to someone else by the current BDFL.
```

Comment:
But then it’s not FL any longer? ;-)

Agree with the concept of a BDFL: members give their trust in your decisions. If they don’t like your decisions, they can still fork the code and go elsewhere. I just wonder whether the BDFL name is a bit outdated and harsh/not-friendly, and whether we’d be able to come up with something else? Could we maybe call it something like “code-base curator”? In the naming we should show that we are working to enable their use cases, and not that there is a dictator that may overrule the board’s decision at any time. There may however be good reasons for a code curator to overrule a board’s decision: it should also be in the interest of how the code is currently being set-up.